set.mm

```diff
@@ -465654,6 +465751,432 @@ its graph has a given second element (that is, function value).
+  cdp2 $a class _ A B $.
+
+  $( Define the "decimal fraction constructor", which is used to build up
+     "decimal fractions" in base 10.  This is intentionally similar to
+     ~ df-dec .  (Contributed by David A. Wheeler, 15-May-2015.)  (Revised by
+     AV, 9-Sep-2021.) $)
+  df-dp2 $a |- _ A B = ( A + ( B / ; 1 0 ) ) $.
+
+  $( Obsolete version of ~ df-dp2 as of 9-Sep-2021.  (Contributed by David A.
+     Wheeler, 15-May-2015.)  (New usage is discouraged.)
+     (Proof modification is discouraged.) $)
+  dfdp2OLD $p |- _ A B = ( A + ( B / 10 ) ) $=
+    ( cdp2 c1 cc0 cdc cdiv co caddc c10 df-dp2 dec10OLD eqcomi oveq2i eqtri ) A
+    BCABDEFZGHZIHABJGHZIHABKQRAIPJBGJPLMNNO $.
+
+  $( Equality theorem for the decimal expansion constructor.  (Contributed by
+     David A. Wheeler, 15-May-2015.) $)
+  dp2eq1 $p |- ( A = B -> _ A C = _ B C ) $=
+    ( wceq c1 cc0 cdc cdiv co caddc cdp2 oveq1 df-dp2 3eqtr4g ) ABDACEFGHIZJIBO
+    JIACKBCKABOJLACMBCMN $.
+
+  $( Equality theorem for the decimal expansion constructor.  (Contributed by
+     David A. Wheeler, 15-May-2015.) $)
+  dp2eq2 $p |- ( A = B -> _ C A = _ C B ) $=
+    ( wceq c1 cc0 cdc cdiv co caddc cdp2 oveq1 oveq2d df-dp2 3eqtr4g ) ABDZCAEF
+    GZHIZJICBQHIZJICAKCBKPRSCJABQHLMCANCBNO $.
+
+  ${
+    dp2eq1i.1 $e |- A = B $.
+    $( Equality theorem for the decimal expansion constructor.  (Contributed by
+       David A. Wheeler, 15-May-2015.) $)
+    dp2eq1i $p |- _ A C = _ B C $=
+      ( wceq cdp2 dp2eq1 ax-mp ) ABEACFBCFEDABCGH $.
+
+    $( Equality theorem for the decimal expansion constructor.  (Contributed by
+       David A. Wheeler, 15-May-2015.) $)
+    dp2eq2i $p |- _ C A = _ C B $=
+      ( wceq cdp2 dp2eq2 ax-mp ) ABECAFCBFEDABCGH $.
+
+    dp2eq12i.2 $e |- C = D $.
+    $( Equality theorem for the decimal expansion constructor.  (Contributed by
+       David A. Wheeler, 15-May-2015.) $)
+    dp2eq12i $p |- _ A C = _ B D $=
+      ( cdp2 dp2eq1i dp2eq2i eqtri ) ACGBCGBDGABCEHCDBFIJ $.
+  $}
+
+  ${
+    dp20u.1 $e |- A e. NN0 $.
+    $( Add a zero in the tenths (lower) place.  (Contributed by Thierry Arnoux,
+       16-Dec-2021.) $)
+    dp20u $p |- _ A 0 = A $=
+      ( cc0 cdp2 c1 cdc cdiv co caddc df-dp2 cc wcel wne 10nn0 nn0rei recni 0re
+      wceq 10pos gtneii div0 mp2an oveq2i nn0cni addid1i 3eqtri ) ACDACECFZGHZI
+      HACIHAACJUHCAIUGKLUGCMUHCRUGUGNOPCUGQSTUGUAUBUCAABUDUEUF $.
+  $}
+
+  ${
+    dp20h.1 $e |- A e. RR+ $.
+    $( Add a zero in the unit places.  (Contributed by Thierry Arnoux,
+       16-Dec-2021.) $)
+    dp20h $p |- _ 0 A = ( A / ; 1 0 ) $=
+      ( cc0 cdp2 c1 cdc cdiv co caddc df-dp2 crp wcel cc ax-mp 10nn0 nn0cni 0re
+      rpcn 10pos gtneii divcli addid2i eqtri ) CADCAECFZGHZIHUECAJUEAUDAKLAMLBA
+      RNUDOPCUDQSTUAUBUC $.
+  $}
+
+  $( Define the closure for the decimal fraction constructor if both values are
+     reals.  (Contributed by David A. Wheeler, 15-May-2015.) $)
```

Comment:
Define the closure for --> Closure of/for
?

Reply:
Agreed! This will be fixed in my next commit.